---
sensor_msgs/RegionOfInterest left
sensor_msgs/RegionOfInterest right
